cpp/include/cudf/column/dlpack.hpp

```diff
@@ -33,19 +35,20 @@ namespace cudf {
  * 
  * @return Columns converted from DLPack
  */
-std::vector<std::unique_ptr<column>> from_dlpack(
-    DLManagedTensor const& managed_tensor,
+std::unique_ptr<experimental::table> from_dlpack(
+    DLManagedTensor const* managed_tensor,
```

Comment:
Why does this need to be a pointer rather than a const reference?

Reply:
Since the `deleter` method of `DLManagedTensor` is supposed to delete the `DLManagedTensor` itself, implying it will always be on the heap, I thought it made more sense to pass it as a poiner. Otherwise, you would always have to explicitly dereference it before passing it by reference.